Orange/widgets/evaluate/owparameterfitter.py

```diff
@@ -252,6 +253,54 @@ def __get_index_at(self, point: QPointF) -> Optional[int]:
         return None
 
 
+class RangePreview(QWidget):
```

Comment:
This is for eliding in the middle of the list, so that preview is shown as, e.g. `10, 20, 30, 40, 50, 60, 70, ..., 120`, not `10, 20, 30, 40, 50, 60, 70, 80, ...`. It also simplifies indenting and aligning.